website/docs/guides/configuration.md

```diff
@@ -23,12 +23,12 @@ scala-cli package --help
 ```
 
 As an example of command line configuration, one thing you can do with `scala-cli` command line options is to specify the Scala version:
-```bash
+```bash ignore
```

Comment:
We shouldn't have so many `ignore` in the documentation, could you create a issue to remove the `ignore` where possible?

Reply:
#1523